### PR TITLE
fix bug of action_mapping

### PR DESCRIPTION
**Pull request by zenghsh3**

### CODE REVIEW

File: parl/env/continuous_wrappers.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        self.high_bound = self.env.action_space.high[0]
+        assert self.high_bound > self.low_bound
+
+    def reset(self, **kwargs):
+        return self.env.reset(**kwargs)
+
+    def step(self, model_output_act):
+        """
+        Args:
+            model_output_act: np.array, which value is in [-1, 1]
```
Comment (TomorrowIsAnOtherDay) [inline parl/env/continuous_wrappers.py]: model_output_act(np.array):  The values must be in in [-1, 1].

File: examples/tutorials/homework/lesson5/ddpg_quadrotor/train.py [hunk 2]
```diff
@@ -104,6 +99,8 @@ def evaluate(env, agent, render=False):
 
 # 创建飞行器环境
 env = make_env("Quadrotor", task="hovering_control")
+env = ActionMappingWrapper(env)
```
Comment (TomorrowIsAnOtherDay) [inline examples/tutorials/homework/lesson5/ddpg_quadrotor/train.py:103]: Please add a comment in Chinese for this line. It's the tutorial for beginners.

File: parl/env/continuous_wrappers.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gym
+import numpy as np
+
+
+class ActionMappingWrapper(gym.Wrapper):
+    def __init__(self, env):
+        """Mapping action space [-1, 1] of model output
```
Comment (TomorrowIsAnOtherDay) [inline parl/env/continuous_wrappers.py]: Mapping -> map

File: parl/env/continuous_wrappers.py (hunk 1 above)
Comment (zenghsh3) [reply parl/env/continuous_wrappers.py]: done

File: examples/tutorials/homework/lesson5/ddpg_quadrotor/train.py (hunk 2 above)
Comment (zenghsh3) [reply examples/tutorials/homework/lesson5/ddpg_quadrotor/train.py:103]: done

File: parl/env/continuous_wrappers.py (hunk 3 above)
Comment (zenghsh3) [reply parl/env/continuous_wrappers.py]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.

**Pull request merged**